examples/component-actions/zarf.yaml

```diff
@@ -211,3 +211,12 @@ components:
                 kind: configmap
                 name: simple-configmap
                 namespace: zarf
+
```

Comment:
A comment stating that `set -e` is the default for commands could be added here or in the `on-create` component that is intro-ing multi line commands (~ line 54)